opentracing/span.py

```diff
@@ -20,45 +20,83 @@
 
 from __future__ import absolute_import
 
-import re
+
+class SpanContext(object):
+    def set_baggage_item(self, key, value):
+        """Stores a Baggage item in the span as a key/value pair.
+
+        Enables powerful distributed context propagation functionality where
+        arbitrary application data can be carried along the full path of
+        request execution throughout the system.
+
+        Note 1: Baggage is only propagated to the future (recursive) children
+        of this SpanContext.
+
+        Note 2: Baggage is sent in-band with every subsequent local and remote
+        calls, so this feature must be used with care.
+
+        :param key: Baggage item key
+        :type key: str
+
+        :param value: Baggage item value
+        :type value: str
+
+        :rtype : SpanContext
+        :return: itself, for chaining the calls.
+        """
+        return self
+
+    def get_baggage_item(self, key):
+        """Retrieves value of the Baggage item with the given key.
+
+        :param key: key of the Baggage item
+        :type key: str
+
+        :rtype : str
+        :return: value of the Baggage item with given key, or None.
+        """
+        return None
```

Comment:
@yurishkuro I thought about that... I decided against it since (a) it's not part of the opentracing.io/spec yet and (b) I wanted to keep the scope of this `SpanContext` change as small as possible. Seems like a great thing to do in a follow-on PR, though.